modules/kuberay-cluster/main.tf

```diff
@@ -31,37 +32,33 @@ resource "helm_release" "ray-cluster" {
   version          = "1.0.0"
   values = [
     var.autopilot_cluster ? templatefile("${path.module}/kuberay-autopilot-values.yaml", {
-      gcs_bucket          = var.gcs_bucket
-      k8s_service_account = var.google_service_account
-      grafana_host        = var.grafana_host
-      security_context    = local.security_context
-      secret_name         = var.db_secret_name
-      project_id          = var.project_id
-      db_region           = var.db_region
+      gcs_bucket                        = var.gcs_bucket
+      k8s_service_account               = var.google_service_account
+      grafana_host                      = var.grafana_host
+      security_context                  = local.security_context
+      secret_name                       = var.db_secret_name
+      cloudsql_instance_connection_name = local.cloudsql_instance_connection_name
```

Comment:
And we still manually type the instance in jupyter hub. will this cause any mis-configuration?

Reply:
no, we don't need to - see the associated notebook change